README.md

```diff
@@ -142,57 +154,46 @@ params = {
 res = requests.post(url, params=params)
 print(res.json())
 ```
+
 **Pip Quickstart**:
 
 After installing via pip, you can run a simple inference using:
 
 ```python
-from inference.core.data_models import ObjectDetectionInferenceRequest
-from inference.models.yolov5.yolov5_object_detection import (
-    YOLOv5ObjectDetectionOnnxRoboflowInferenceModel,
-)
+from inference.models.utils import get_roboflow_model
 
-model = YOLOv5ObjectDetectionOnnxRoboflowInferenceModel(
-    model_id="soccer-players-5fuqs/1", device_id="my-pc", 
+model = get_roboflow_model(
```

Comment:
Could you also add CLIP and SAM examples here too (assuming they are 2-3 lines of code each like these ones)? Maybe in separate example code blocks like the supervision README.

Reply:
Yes, I'll do that in a "soon to follow" PR where I also add extras to our pip install for clip and sam (by replacing their github repos in our requirements files with the roboflow hosted pypi versions)